includes/controls/groups/border.php

```diff
@@ -73,12 +73,14 @@ protected function init_fields() {
 			],
 		];
 
+		// To do: remove after we update all widget border-width properties to logical.
+		$border_physical_properties = 'border-width: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};';
```

Comment:
Fallback value for widgets in core and pro that don't yet implement logical Border.

Reply:
I think that we should remove this.
As far as I can see, this control doesn't use a CSS variable but will general the CSS properties directly. I don't think there is a need to create physical and logical properties at the same time.

Reply:
Mega Menu overrides border-width in the control. Other Nested Elements use logical defaults set in the border control.

There are several widgets beyond the scope of this PR that we need to check when we flip to logical CSS on the border control. This is the reason I'm proposing fallback values until we have updated tested. 

### Widgets possibly affected:

#### Core:
- testimonial
- column
- container
- section
- common
- image-carousel
- image-gallery
- image
- social-icons
- testimonial
- button (button trait)

#### Pro:
- blockquote
- call-to-action
- countdown
- flip-box
- form
- login
- taxonomy-filter
- nav-menu
- popup
- price-table
- Theme Builder
- Woocommerce & Woocommerce widgets
- popup